internal/appsec/_tools/rules-updater/escaper/go.mod

```diff
@@ -1,3 +1,3 @@
 module escaper
 
-go 1.16
+go 1.17
```

Comment:
This is an internal tool we use to update the ASM security rules (`internal/appsec/rule.go`), bumping is fine but not required 👍